tests/testing/LocalBearTestHelperTest.py

```diff
@@ -0,0 +1,43 @@
+from queue import Queue
+import unittest
+
+from tests.test_bears.TestBear import (FirstTestBear,
+                                       SecondTestBear,
+                                       ThirdTestBear)
+from coalib.settings.Section import Section
+from coalib.settings.Setting import Setting
+from coalib.testing.LocalBearTestHelper import verify_local_bear, execute_bear
+
+
+files = (("Everything is invalid/valid/raises error",),)
```

Comment:
Why not initialize these inside the class `setUp` function? IMO that's what this is for. Or are you gonna create another class that will reuse this in the future?